wolfcrypt/src/sha512.c

```diff
@@ -1618,8 +1622,14 @@ int wc_Sha512GetFlags(wc_Sha512* sha512, word32* flags)
 #if !defined(WOLFSSL_NOSHA512_224)
 int wc_InitSha512_224(wc_Sha512* sha)
 {
+    if (sha == NULL)
```

Comment:
This is redundant. Already done in `wc_InitSha512_224_ex` -> `InitSha512_Family`.

Reply:
agreed. good catch. updated.